src/main/java/mekanism/common/registries/MekanismModules.java

```diff
@@ -59,7 +60,9 @@ private MekanismModules() {
     public static final ModuleRegistryObject<ModuleShearingUnit> SHEARING_UNIT = MODULES.register("shearing_unit", ModuleShearingUnit::new,
           () -> MekanismItems.MODULE_SHEARING.asItem(), builder -> builder.rarity(Rarity.UNCOMMON).exclusive(ExclusiveFlag.INTERACT_ENTITY, ExclusiveFlag.INTERACT_BLOCK));
     public static final ModuleRegistryObject<ModuleSilkTouchUnit> SILK_TOUCH_UNIT = MODULES.register("silk_touch_unit", ModuleSilkTouchUnit::new,
-          () -> MekanismItems.MODULE_SILK_TOUCH.asItem(), builder -> builder.rarity(Rarity.RARE));
+          () -> MekanismItems.MODULE_SILK_TOUCH.asItem(), builder -> builder.rarity(Rarity.RARE).exclusive(ExclusiveFlag.OVERRIDE_DROPS));
+    public static final ModuleRegistryObject<ModuleFortuneUnit> FORTUNE_UNIT = MODULES.register("fortune_unit", ModuleFortuneUnit::new,
+          () -> MekanismItems.MODULE_FORTUNE.asItem(), builder -> builder.maxStackSize(4).rarity(Rarity.RARE).exclusive(ExclusiveFlag.OVERRIDE_DROPS));
```

Comment:
Max stack size should probably be 3, I don't see a reason to allow going above the max vanilla fortune level.

Reply:
Well I did 4 cause I didn't want to make a stack size that wasn't a power of 2. I also didn't really see a problem with 4 since most mekanism stuff is better than vanilla anyway and it doesn't really need to relate to vanilla fortune. But if you feel strongly about it I can change it.

Reply:
This doesn't really have anything to do with this pr or 1.18 release, but what do you think about having configs for some of the stack sizes. It'd be trivial for this one, but still doable to have a range for some like vein mining, blasting, efficiency, attack damage. And I could actually see someone deciding blasting should only go to 5x5 or something like that.